src/clusterfuzz/_internal/bot/tasks/task_types.py

```diff
@@ -82,6 +82,12 @@ def is_remote_utask(command, job):
   return batch.is_remote_task(command, job)
 
 
+def is_untrusted_task(command, job):
```

Comment:
nit: is_untrusted_task feels a bit confusing to me in this context. It's not clear what being an untrusted task means.

Would it make more sense to rename this as "handles_untrusted", which makes it clearer that this is checking if a particular workload can handle untrusted testcases.